torch/_dynamo/variables/user_defined.py

```diff
@@ -326,6 +326,10 @@ def var_getattr(self, tx, name):
             )
         elif isinstance(subobj, types.FunctionType):
             return variables.UserMethodVariable(subobj, self, source=source, **options)
+        elif hasattr(subobj, "__get__"):
```

Comment:
Alternately, we could dispatch on `@isinstance(.., lazy_property)`, it's definitely safer, I think this is not sound